scripts/flags/flags.js

```diff
@@ -375,53 +381,86 @@ const FLAG_CONFIG = {
 
 const FLAG_COLUMNS = Object.keys(FLAG_CONFIG);
 
+const INTERNAL_VARIANTS = ['WWW Classic', 'WWW Modern', 'RN OSS'];
```

Comment:
Should this be RN FB?
```suggestion
const INTERNAL_VARIANTS = ['WWW Classic', 'WWW Modern', 'RN FB'];
```

Reply:
Yeah, updated